qutebrowser/misc/earlyinit.py

```diff
@@ -243,6 +243,11 @@ def check_libraries():
     if sys.version_info < (3, 9):
         # Backport required
         modules['importlib_resources'] = _missing_str("importlib_resources")
+    from qutebrowser.utils import utils
```

Comment:
We shouldn't import from `qutebrowser.utils` at this point yet, as that would fail (before the user-facing error message is shown below) if e.g. PyQt wasn't available. I suppose it's best to just use `sys.platform` here.